sonar-php-plugin/src/test/java/org/sonar/plugins/php/reports/phpunit/CoverageFileParserForPhpUnitTest.java

```diff
@@ -39,23 +39,25 @@ public class CoverageFileParserForPhpUnitTest {
 
   private CoverageFileParserForPhpUnit parser;
 
-  @Before
+  @BeforeEach
   public void setUp() {
     parser = new CoverageFileParserForPhpUnit();
   }
 
   @Test
-  public void should_parse_when_there_is_no_project() throws IOException {
+  void shouldParseWhenThereIsNoProject() throws IOException {
     CountConsumer counter = new CountConsumer();
     parser.parse(reportFile(NO_PROJECT_FILE), counter);
     assertThat(counter.count).isZero();
   }
 
   @Test
-  public void should_fail_when_xml_root_node_is_not_coverage() {
+  void shouldFailWhenXmlRootNodeIsNotCoverage() {
```

Comment:
Nitpick: shouldn't it be `Covered` instead of `Coverage`?

Reply:
Regex didn't catch that ;-)